packages/type-metadata/src/consts/fromMetadata/fromMetadata.spec.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017-2019 @polkadot/metadata authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Metadata, BlockNumber } from '@polkadot/types';
+import json from '@polkadot/types/Metadata/v6/static';
+
+import fromMetadata from '../fromMetadata';
+
+// Use the pre-generated metadata
+const metadata = new Metadata(json);
+const consts = fromMetadata(metadata);
+
+describe('fromMetadata', () => {
+  it('should return constants with the correct type and value', () => {
+    expect(consts.democracy.enactmentPeriod).toBeInstanceOf(BlockNumber);
+    expect(consts.democracy.enactmentPeriod.toHex(true)).toEqual('0x80f4030000000000');
```

Comment:
Probably better here to have `0x80f4030000000000` as a decimal number? (i didn't find it in the rust code...)

Reply:
https://github.com/paritytech/substrate/blob/14b15fc4c600c046aa0d172d2d5e045134a40c3b/node/runtime/src/lib.rs#L233

`latest.substrate.v6.json` isn't up-to-date though (has an old value)

Reply:
Ah I see. So we could
- finish #1067 and update static
- and in that PR, update this test so that the value is exactly the one defined in rust

mental note to self to check that in the next PR